js/repl/PluginConfig.js

```diff
@@ -1,11 +1,57 @@
 // @flow
+import camelCase from "lodash.camelcase";
+import type { PluginConfig, MultiPackagesConfig, EnvFeatures } from "./types";
 
-import type { PluginConfig } from "./types";
+const convertPluginNameToInsanceName = pluginName =>
+  `_babel_${camelCase(`plugin-${pluginName}`)}`;
```

Comment:
I assumed that you meant `instance` here but it doesn't look very clear to me.
What about `normalize`, `toBabelPluginName`, `getFullyQualifiedPluginName`?

I can see that we can save a few CPU cycles (and thus probably trees :earth_africa: ) here, `plugin-` is being camelCase'd every time even if it's a constant.

Reply:
> I assumed that you meant instance here but it doesn't look very clear to me.
What about normalize, toBabelPluginName, getFullyQualifiedPluginName

Sure! 👍

Reply:
> I can see that we can save a few CPU cycles (and thus probably trees 🌍 ) here, plugin- is being camelCase'd every time even if it's a constant.

The other way is `_babel_plugin${capitalize(camelCase(pluginName))}`.
I don't think it is more performant and need an additional dependency.